### PR TITLE
Make integration test output human-readable

### PR DESCRIPTION
**Pull request by vlad-timofeev**
# Overview

Make integration test output human-readable. Related to #436.

# Description

1. Change logger configuration of integration tests to treat `\n` as new lines.
2. Do not compact JSONs from PUT request for readability. 

# How Has This Been Tested?

Ran `mage -v integration` and visually inspected test output to confirm that readability has improved.

# Checklist

Before submitting this PR, please make sure:

- [x] I have read the CONTRIBUTING document.
- [x] My code is consistent with the rest of the project 
- [x] I have tagged the relevant reviewers and/or interested parties
- [x] I have updated the READMEs and other documentation of affected packages

## References

Related issue https://github.com/sirupsen/logrus/issues/608

### CODE REVIEW

**Review by nitro-neal — APPROVED** [review]
Looks good, do you have some example output you can paste?

**Comment by vlad-timofeev** [comment]
Yes, this is the same test output as given in #436 , but with the code change applied.

```
=== RUN   TestDIDWebCreateVerifiableCredentialIntegration
time=2023-05-19T14:39:08-04:00 level=info msg=

Create a verifiable credential
time=2023-05-19T14:39:08-04:00 level=info msg=
Performing PUT request to:  http://localhost:8080/v1/credentials

with data:
{
  "data": {
    "givenName":"ricky bobby",
    "additionalName":"hank hill",
    "familyName":"simpson",
    "birthDate":"2009-01-03",
    "postalAddress": {
      "addressCountry": "U.S.A",
      "addressLocality": "Austin",
      "addressRegion": "TX",
      "postalCode": "78724",
      "streetAddress": "123 Janktopia Ave."
    },
    "taxID":"123"
  },
  "issuer": "did:web:tbd.website",
  "issuerKid": "did:web:tbd.website#owner",
  "subject": "did:web:tbd.website",
  "@context":"https://www.w3.org/2018/credentials/v1",
  "expiry": "2051-10-05T14:48:00.000Z",
  "schemaId": "290e5d6c-cdd1-4e33-aa12-5f3029df40cd"
}

time=2023-05-19T14:39:08-04:00 level=info msg=
Output:
time=2023-05-19T14:39:08-04:00 level=info msg={
    "credential": {
        "@context": [
            "https://www.w3.org/2018/credentials/v1"
        ],
        "id": "3145b64e-a59e-43e5-8ac7-9c58c0506468",
        "type": [
            "VerifiableCredential"
        ],
        "issuer": "did:web:tbd.website",
        "issuanceDate": "2023-05-19T18:39:08Z",
        "expirationDate": "2051-10-05T14:48:00.000Z",
        "credentialSubject": {
            "additionalName": "hank hill",
            "birthDate": "2009-01-03",
            "familyName": "simpson",
            "givenName": "ricky bobby",
            "id": "did:web:tbd.website",
            "postalAddress": {
                "addressCountry": "U.S.A",
                "addressLocality": "Austin",
                "addressRegion": "TX",
                "postalCode": "78724",
                "streetAddress": "123 Janktopia Ave."
            },
            "taxID": "123"
        },
        "credentialSchema": {
            "id": "290e5d6c-cdd1-4e33-aa12-5f3029df40cd",
            "type": "JsonSchemaValidator2018"
        }
    },
    "credentialJwt": "eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDp3ZWI6dGJkLndlYnNpdGUjb3duZXIiLCJ0eXAiOiJKV1QifQ.eyJleHAiOjI1ODAxMzAwODAsImlhdCI6MTY4NDUyMTU0OCwiaXNzIjoiZGlkOndlYjp0YmQud2Vic2l0ZSIsImp0aSI6IjMxNDViNjRlLWE1OWUtNDNlNS04YWM3LTljNThjMDUwNjQ2OCIsIm5iZiI6MTY4NDUyMTU0OCwibm9uY2UiOiI3MDdjZWIwMi0xMDVkLTQwYTYtYWY0NS00MWQ3NjMxZWFkNGMiLCJzdWIiOiJkaWQ6d2ViOnRiZC53ZWJzaXRlIiwidmMiOnsiQGNvbnRleHQiOlsiaHR0cHM6Ly93d3cudzMub3JnLzIwMTgvY3JlZGVudGlhbHMvdjEiXSwidHlwZSI6WyJWZXJpZmlhYmxlQ3JlZGVudGlhbCJdLCJpc3N1ZXIiOiIiLCJpc3N1YW5jZURhdGUiOiIiLCJjcmVkZW50aWFsU3ViamVjdCI6eyJhZGRpdGlvbmFsTmFtZSI6ImhhbmsgaGlsbCIsImJpcnRoRGF0ZSI6IjIwMDktMDEtMDMiLCJmYW1pbHlOYW1lIjoic2ltcHNvbiIsImdpdmVuTmFtZSI6InJpY2t5IGJvYmJ5IiwicG9zdGFsQWRkcmVzcyI6eyJhZGRyZXNzQ291bnRyeSI6IlUuUy5BIiwiYWRkcmVzc0xvY2FsaXR5IjoiQXVzdGluIiwiYWRkcmVzc1JlZ2lvbiI6IlRYIiwicG9zdGFsQ29kZSI6Ijc4NzI0Iiwic3RyZWV0QWRkcmVzcyI6IjEyMyBKYW5rdG9waWEgQXZlLiJ9LCJ0YXhJRCI6IjEyMyJ9LCJjcmVkZW50aWFsU2NoZW1hIjp7ImlkIjoiMjkwZTVkNmMtY2RkMS00ZTMzLWFhMTItNWYzMDI5ZGY0MGNkIiwidHlwZSI6Ikpzb25TY2hlbWFWYWxpZGF0b3IyMDE4In19fQ.xO-nko2KG84ntg6bZZwm9wKzF12FbODrRrC1O6sNXQdPHm6pjgYaNnOWpVyJDTNmIevZwlcBukEmcQr_F0h1DA"
}
--- PASS: TestDIDWebCreateVerifiableCredentialIntegration (0.00s)
```

File: integration/common.go [hunk 1]
```diff
@@ -379,7 +386,7 @@ func get(url string) (string, error) {
 }
 
 func put(url string, json string) (string, error) {
-	logrus.Println(fmt.Sprintf("\nPerforming PUT request to:  %s \n\nwith data: \n%s\n", url, compactJSONOutput(json)))
+	logrus.Println(fmt.Sprintf("\nPerforming PUT request to:  %s \n\nwith data: \n%s\n", url, json))
```
Comment (decentralgabe) [inline integration/common.go:389]:
```suggestion
	logrus.Printf("\nPerforming PUT request to:  %s \n\nwith data: \n%s\n", url, json)
```

**Comment by decentralgabe** [comment]
tests failing due to my change - fix here https://github.com/TBD54566975/ssi-service/pull/445

**Comment by vlad-timofeev** [comment]
Thanks for super quick review and feedback!

**Pull request merged**